### ISSUE TITLE
How do we know which all features are being used in multi "regressor" mode ?

### ISSUE THREAD

**Issue by vinitkothari24** [winedarksea/AutoTS#166]
How do we know which all features are being used in multi "regressor" mode ?

**Comment by winedarksea** [comment]
You don't.
It really depends no the model that is chosen. Some models you can extract the information, but for most you can't. Cassandra has coefficients. VAR will soon have pvalues accessible. It should be possible, if convoluted, to extract feature_importances_ from some of the ~Regression models.
Probably the simplest way, and only way for some models, is to extract the best_model and run model_forecast both with and without a certain feature in future_regressor to see if and how much it makes a difference.

**Comment by El-Don-Quijote** [comment]
Really looking forward to VAR p-values!

**Comment by winedarksea** [comment]
@El-Don-Quijote it should be available in 0.5.4 the latest release, try testing it out.
Here is an example. Note you should be able to access any of the details of the statsmodels VAR result. This should also work with results from AutoTS.predict, not just model_forecast.
```python
import numpy as np
import pandas as pd
import matplotlib.pyplot as plt
from autots import model_forecast, load_monthly


df = load_monthly(long=False)
forecast_length = 8
df_train = df.iloc[:-forecast_length]
df_test = df.iloc[-forecast_length:]

from autots.models.statsmodels import VAR
params = VAR().get_new_params(method='fast')
print(params)
forecasts = model_forecast(
    model_name="VAR",
    model_param_dict=params,
    model_transform_dict={'fillna': 'ffill'},
    df_train=df_train,  # .iloc[:, 0:1]
    forecast_length=forecast_length,
    frequency='infer',
    prediction_interval=0.9,
    no_negatives=False,
    future_regressor_train=None,
    future_regressor_forecast=None,
    random_seed=321,
    verbose=1,
    n_jobs=4,
    return_model=True,  # THIS IS IMPORTANT FOR GETTING DETAILS
    fail_on_forecast_nan=True,
)
series = 'wiki_all'
series = series if series in df.columns else df.columns[0]
forecasts.plot(df_wide=df_train, series=series, start_date=df_train.index.max() - pd.Timedelta(days=365))
forecasts.model.model.summary()  # this is Statsmodels statistical summary
```
